crest/Assets/Crest/Crest/Scripts/Shapes/ShapeGerstner.cs

```diff
@@ -159,6 +170,9 @@ struct GerstnerWaveComponent4
             public Vector4 _omega;
             public Vector4 _phase;
             public Vector4 _chopAmp;
+            public Vector4 _amp2;
+            public Vector4 _chopAmp2;
+            public Vector4 _phase2;
```

Comment:
If not too much trouble, adding a note here that these are for wave pairs will make it easier to understand their purpose (at least for me). And maybe some other places where you feel appropriate.

Reply:
thanks, done